panic in pgbouncer plugin

### Relevant telegraf.conf:
<!-- [Place config betweetn triple backticks '```'] -->
```toml
[[inputs.pgbouncer]]
  ## specify address via a url matching:
  ##   postgres://[pqgotest[:password]]@localhost[/dbname]\
  ##       ?sslmode=[disable|verify-ca|verify-full]
  ## or a simple string:
  ##   host=localhost user=pqotest password=... sslmode=... dbname=app_production
  ##
  ## All connection parameters are optional.
  ##
  address = "host=localhost port=6432 user=dbuser database=pgbouncer"
```

### System info:

Debian 8, telegraf 1.11.0-1, pgbouncer 1.5.4

### Steps to reproduce:
1. Enable pgbouncer input

### Expected behavior:
Telegraf starts to report data

### Actual behavior:
Telegraf is not able to start

### Additional info:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x15ac136]
goroutine 43 [running]:
github.com/influxdata/telegraf/plugins/inputs/pgbouncer.(*PgBouncer).Gather(0xc000556cd0, 0x2a32740, 0xc000223660, 0x0, 0x0)
/go/src/github.com/influxdata/telegraf/plugins/inputs/pgbouncer/pgbouncer.go:105 +0x746
github.com/influxdata/telegraf/internal/models.(*RunningInput).Gather(0xc000139300, 0x2a32740, 0xc000223660, 0x2a17b80, 0xc000093ec0)
/go/src/github.com/influxdata/telegraf/internal/models/running_input.go:86 +0x6d
github.com/influxdata/telegraf/agent.(*Agent).gatherOnce.func1(0xc00027c0c0, 0xc000139300, 0x2a32740, 0xc000223660)
/go/src/github.com/influxdata/telegraf/agent/agent.go:299 +0x3f
created by github.com/influxdata/telegraf/agent.(*Agent).gatherOnce
/go/src/github.com/influxdata/telegraf/agent/agent.go:298 +0xdc
```

--- a/plugins/inputs/pgbouncer/pgbouncer.go
+++ b/plugins/inputs/pgbouncer/pgbouncer.go
@@ -2,14 +2,12 @@ package pgbouncer
 
 import (
 	"bytes"
-	"github.com/influxdata/telegraf/plugins/inputs/postgresql"
-
-	// register in driver.
-	_ "github.com/jackc/pgx/stdlib"
 
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/internal"
 	"github.com/influxdata/telegraf/plugins/inputs"
+	"github.com/influxdata/telegraf/plugins/inputs/postgresql"
+	_ "github.com/jackc/pgx/stdlib" // register driver
 )
 
 type PgBouncer struct {
@@ -98,12 +96,16 @@ func (p *PgBouncer) Gather(acc telegraf.Accumulator) error {
 			return err
 		}
 
-		if s, ok := (*columnMap["user"]).(string); ok && s != "" {
-			tags["user"] = s
+		if user, ok := columnMap["user"]; ok {
+			if s, ok := (*user).(string); ok && s != "" {
+				tags["user"] = s
+			}
 		}
 
-		if s, ok := (*columnMap["pool_mode"]).(string); ok && s != "" {
-			tags["pool_mode"] = s
+		if poolMode, ok := columnMap["pool_mode"]; ok {
+			if s, ok := (*poolMode).(string); ok && s != "" {
+				tags["pool_mode"] = s
+			}
 		}
 
 		fields := make(map[string]interface{})

